nomos-da/kzgrs-backend/src/verifier.rs

```diff
@@ -131,7 +109,18 @@ impl DaVerifier {
         let message =
             build_attestation_message(&blob.aggregated_column_commitment, &blob.rows_commitments);
         let signature = self.sk.sign(&message, b"", b"");
-        Attestation { signature }
+
+        let blob_id = blob.id();
+        let blob_hash: [u8; 32] = blob_id
+            .try_into()
+            .expect("Blob ID must be exactly 32 bytes long");
+
+        Attestation {
+            signature: signature.to_bytes().to_vec(),
+            // TODO: Remove if not required, or implement serialization for 96 bytes pub key.
+            attester: [0u8; 32],
```

Comment:
This is probably not needed? The one receiving attestation should know how is answering. Usually you do not want to check a signature over a pk that comes in the same message.